lib/community/widgets/post_card_list.dart

```diff
@@ -194,7 +194,7 @@ class _PostCardListState extends State<PostCardList> with TickerProviderStateMix
                       },
                       child: CommunityHeader(communityInfo: widget.communityInfo),
                     );
-                  } else if (widget.tagline?.isNotEmpty == true) {
+                  } else if (widget.tagline!.isNotEmpty) {
```

Comment:
It looks like due to [this line](https://github.com/thunder-app/thunder/blob/736d741129ce2c0d33046acd66efbd32cd75efe5/lib/community/bloc/community_bloc.dart#L266), `tagline` can never be `null`. So we should probably go through and make it not nullable. Otherwise, this looks like a code smell (null assertion operator without null check). But fine for now.

Reply:
Ah, I tested that but did not leave it in... I think I committed in-between leaving it `tagline!`

Reply:
Restored, I could go and do that while I'm at it...

Reply:
Hey, sorry, I think I was unclear! At first I thought something was fishy because you used `!` without checking for `null`. But after I investigated where `tagline` is set ([here](https://github.com/thunder-app/thunder/blob/736d741129ce2c0d33046acd66efbd32cd75efe5/lib/community/bloc/community_bloc.dart#L266)), I realized it's actually never `null`, so your null assertion would be safe. I only said it's a code smell because other people might have a similar thought (how do we know it's safe to use null assertion without checking). So I think the better solution would be to make the type non-null entirely (`String` instead of `String?`). Then we wouldn't need either the null assertion or null conditional operator here.

Let me know if that makes sense and if you want to make that change. If not, I'll go ahead and merge this and do it later. Your code is totally fine either way, it just got me thinking. 😆